.gitignore

```diff
@@ -12,3 +12,4 @@
 # Avoid checking in db files
 *.db
 *.json
+.vscode
```

Comment:
i think this directory is intended to be checked in

Reply:
https://www.toptal.com/developers/gitignore/api/visualstudiocode is one recommendation, if you're getting things in .vscode that you don't want to check in.

Reply:
I'll add the ignores @DaveyWood suggests. I had excluded the folder so as not to bias future contributors and their IDEs :D